aircraft/f-14b/Nasal/awg_9.nas

```diff
@@ -347,6 +347,7 @@ var az_scan = func(notification) {
 # be ok; the values (distance etc) will be read from the target list so these will be accurate
 # which isn't quite how radar works but it will be good enough for us.
 
+    setprop("sim/multiplay/visibility-range-nm", 200);
```

Comment:
In theory this is fine however it is sufficient to do this during startup and not every frame in the radar loop. I suggest moving it into the FDM initialisation.

Reply:
Makes sense, will do.

Reply:
![image](https://user-images.githubusercontent.com/2045654/63871783-d240af80-c9bc-11e9-875b-f996d81bc2b8.png)